library/src/main/java/com/getstream/sdk/chat/view/activity/AttachmentDocumentActivity.java

```diff
@@ -62,7 +64,20 @@ private void configUIs() {
      */
     public void loadDocument(String url) {
         progressBar.setVisibility(View.VISIBLE);
-        webView.loadUrl("https://docs.google.com/gview?embedded=true&url=" + StreamChat.getInstance(this).getUploadStorage().signFileUrl(url));
+        Client client = null;
+        BaseStorage storage = null;
+        String fileUrl = null;
+
+        try {
+            client = StreamChat.getInstance(this);
+            storage = client.getUploadStorage();
+            fileUrl = storage.signFileUrl(url);
+            webView.loadUrl("https://docs.google.com/gview?embedded=true&url=" + StreamChat.getInstance(this).getUploadStorage().signFileUrl(url));
+        } catch (NullPointerException ex) {
```

Comment:
That's a workaround. There is clear reason for NPE: `getUploadStorage` returns null. So there could just basic check:
```kotlin
if(StreamChat.getInstance(this). getUploadStorage() == null){
  //load url
} else{
  finish();
}
```